### DISCUSSION TITLE
Claiming something was already discovered in ancient India, a case study. | Soumik Mukhopadhyay

### DISCUSSION THREAD

**Discussion by giscus** [alshedivat/al-folio#2691] [Comments]
# Claiming something was already discovered in ancient India, a case study. | Soumik Mukhopadhyay

https://soumik-kanad.github.io/blog/2023/ancient-india-and-newtons-laws/

**Comment by raheja88** [comment]
https://www.manchester.ac.uk/about/news/indians-predated-newton-discovery-by-250-years/

**Reply by soumik-kanad** [reply]
This is the same article that the video uses for its basis. So, use if with a grain of salt. 

I read through the article and it mostly talks about math (while I was talking about physics), and the fact that there have been Indian mathematicians like Madhava from the Kerala school who pioneered a bunch of infinite series and initial versions of calculus. All of this seems to be valid [according to Wikipedia](https://en.wikipedia.org/wiki/Madhava_of_Sangamagrama). 

The problem arises when Indian creators like the video I shared above say "Manchester University ne confirm kar diya ki Issac Newton ne Calculus aur Gravity ki theory Bharatiya kitabo se copy ki hai". While the article actually says "The brilliance of Newton's work at the end of the seventeenth century stands undiminished - especially when it came to the algorithms of calculus." 

Secondly, the article itself seems to be mostly an opinion piece and tries to convince that the 'Kerala School' should be attributed as the discoverer of calculus by quoting Dr George Gheverghese Joseph's statements. Most of Dr. Joseph's statements seem valid but their interpretation by the author of the article is flawed and lacks any citation. 

So, to me Madhava's contributions are quite pleasantly surprising and should be recognised where they are due. But that cannot be equated to the impact that Newton's (or Leibniz's) calculus had on modern science and math.

**Comment by UniInc** [comment]
Wow! 😮 Tell us more?
